### PR TITLE
jetty-java8:  Copy Jetty files and the way Tomcat installs Java 8

### PR DESCRIPTION
**Pull request by juharris**

### CODE REVIEW

**Comment by juharris** [comment]
I'll add these to Docker Hub so we can wait until they build there before merging.

**Comment by Mmdixon** [comment]
download link fails. The new some seems to be `http://download.eclipse.org/jetty/8.1.17.v20150415/dist/jetty-distribution-8.1.17.v20150415.tar.gz`

**Comment by Mmdixon** [comment]
This has been failing on our jetty java7 ones for the last 3 months

File: docker-jetty-java8-download/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+RUN apt-get -qq update
+RUN apt-get -qq install software-properties-common python-software-properties
+
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0x219BD9C9
+RUN apt-add-repository "deb http://repos.azulsystems.com/ubuntu stable main"
+RUN add-apt-repository -y ppa:ubuntu-toolchain-r/test
+RUN apt-get -qq update
+RUN apt-get -qq upgrade
+
+RUN apt-get -qq install openjdk-7-jre curl bash unzip
```
Comment (Mmdixon) [inline docker-jetty-java8-download/Dockerfile]: is this java7 dependency still needed?

Comment (juharris) [inline docker-jetty-java8-download/Dockerfile]: Let's find out!

Justin Harris | NLP
Justin@Maluuba.com

**Comment by Mmdixon** [comment]
LGTM

**Pull request merged**